packages/sanity/src/ssr-test/strict-mode-tests.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
```

Comment:
Do you plan to combine strict mode tests and hydration tests? The files look almost identical.